test/test_modified_scripts.py

```diff
@@ -108,13 +95,13 @@ def install_modified():
                   'table_name': '{db}.{table}'},
 
         "xml": {'engine': 'xml',
-                'table_name': 'output_file_{table}.xml'},
+                'table_name': '{db}_{table}.xml'},
```

Comment:
@henrykironde is this formatting ok? I know we had trouble in the past with some output names but I'm not knowledgeable on the current status of how this works.

Reply:
You should checkout the format from https://github.com/weecology/retriever/blob/961974cb18940e26d5dd86f5f04ed157d08b3c7f/test/test_integration.py#L546

Reply:
ok just double checking, this looks about right